test/annotation_utils.t

```diff
@@ -7,29 +7,31 @@
 use Config::Simple;
 use Bio::KBase::AuthToken;
 
+use RASTTestUtils;
+
 use installed_clients::WorkspaceClient;
 use installed_clients::GenomeFileUtilClient;
 use RAST_SDK::RAST_SDKImpl;
 
-use strict;
-use_ok "anno_utils";
-use testRASTutil;
-
+use_ok "RAST_SDK::AnnotationUtils";
 
 ## global variables
-my $token = $ENV{'KB_AUTH_TOKEN'};
-my $config_file = $ENV{'KB_DEPLOYMENT_CONFIG'};
-my $config = new Config::Simple($config_file)->get_block('RAST_SDK');
-my $auth_token = Bio::KBase::AuthToken->new(
-        token => $token, ignore_authrc => 1, auth_svc=>$config->{'auth-service-url'});
-my $ws_url = $config->{"workspace-url"};
-my $ws = get_ws_name();
-my $ws_client = new installed_clients::WorkspaceClient($ws_url,token => $token);
+my $token       = $ENV{ 'KB_AUTH_TOKEN' };
+my $config_file = $ENV{ 'KB_DEPLOYMENT_CONFIG' };
+my $config      = Config::Simple->new( $config_file )->get_block( 'RAST_SDK' );
+my $auth_token  = Bio::KBase::AuthToken->new(
+    token         => $token,
+    ignore_authrc => 1,
+    auth_svc      => $config->{ 'auth-service-url' }
+);
+
+my $ws_client     = RASTTestUtils::get_ws_client();
+my $ws_name       = RASTTestUtils::get_ws_name();
```

Comment:
rename `$ws` to `$ws_name` throughout, since it is a better descriptor of what the entity is (the workspace name)